packages/app-builder/src/routes/_builder+/scenarios+/$scenarioId+/i+/$iterationId+/_edit-view+/decision.tsx

```diff
@@ -54,16 +56,22 @@ function getFormSchema(t: TFunction<typeof handle.i18n>) {
         .number({
           message: t('scenarios:validation.decision.score_threshold_missing'),
         })
+        .max(MAX_THRESHOLD)
+        .min(-MAX_THRESHOLD)
```

Comment:
Everything is good, you just need to pass a second arg on each `max` / `min` to handle translations (you can take inspiration from the message above).

NB: try to inject the `MAX_THRESHOLD` inside the translation

Reply:
thanks, will do next week